### DISCUSSION TITLE
Need libm.so for tflite_runtime

### DISCUSSION THREAD

**Discussion by Vojtak42** [termux/termux-packages#11382] [Q&A] [answered]
Hello,
I need libm.so for tflite_runtime.
`ImportError: dlopen failed: library "libm.so.6" not found: needed by /data/data/com.termux/files/usr/lib/python3.9/site-packages/tflite_runtime/_pywrap_te`
How can I get libm.so?
Thank you for answer.

**Comment by licy183** [comment, marked as answer]
It seems that `tflite_runtime` is not compiled under Termux but under GNU/Linux, so this package is not available on Termux.

By the way, tensorflow has already had a package request (https://github.com/termux/termux-packages/issues/784).
